app/forms/register_nonprofit_form/nonprofit_form.rb

```diff
@@ -1,54 +1,51 @@
 # License: AGPL-3.0-or-later WITH Web-Template-Output-Additional-Permission-3.0-or-later
+
 class RegisterNonprofitForm::NonprofitForm < ApplicationForm
   attribute :name, :string, default: "" # needed because OnboardAccounts is goofy
 
-  attr_accessor :city, 
-    :email,
-    :state_code,
-    :phone,
-    :website,
-    :zip_code
+  attr_reader :website
```

Comment:
What is this one only `attr_reader`?

Reply:
The linter was complaining that it was attr_accessor and also a shadow `def website=` method below.

Reply:
Hmm, I installed standard temporarily just to understand all of the changes but I might not have installed the exact same packages as you. I guess I can see what it's saying. It's probably technically slightly more performant this way I suppose.